solarforecastarbiter/reports/templates/html/macros.j2

```diff
@@ -50,6 +50,59 @@ forecasts/single/{{ forecast.forecast_id }}
 {% endmacro %}
 
 
+{% macro summary_stats_table_vert(summary_stats, category) %}
+{% set span = (human_statistics.values() | length) %}
+<div class="report-table-wrapper">
+<table class="table table-striped table-bordered summary-stats-table-vert" style="width:100%;">
+  <caption style="caption-side:top; text-align: left">
+    Table of {{ human_categories[category] }} data summary statistics
+  </caption>
+  <thead>
+    <tr class="header">
+      <th scope="col" style="text-align: left;">Aligned Pair</th>
+      <th scope="col" colspan="{{ span }}" style="text-align: left;">Observation</th>
+      <th scope="col" colspan="{{ span }}" style="text-align: left;">Forecast</th>
+      <th scope="col" colspan="{{ span }}" style="text-align: left;">Reference Forecast</th>
```

Comment:
I think align center would be more consistent with the other tables, especially the "Table of data alignment parameters". Not totally opposed to align left.